Gist/Managers/MessageQueueManager.swift

```diff
@@ -2,61 +2,69 @@ import Foundation
 
 class MessageQueueManager {
     private var queueTimer: Timer!
-
+    
     func setup() {
         queueTimer = Timer.scheduledTimer(timeInterval: 10,
                                           target: self,
                                           selector: #selector(checkForMessages),
                                           userInfo: nil,
                                           repeats: true)
-        checkForMessages()
+        
+        // Wait 1 second for app to become active.
```

Comment:
I want to suggest adding a little more notes to the comment suggesting why this change was introduced. 

What was happening before that adding a delay helps resolve?

Reply:
I added more context in the comments.

On application launch, there's a short period where the app still shows as in "background" and since we're now checking the state before fetching the queue, the first check was always skipped.